web/src/site-admin/externalServices.tsx

```diff
@@ -335,7 +335,7 @@ export const ALL_EXTERNAL_SERVICES: Record<GQL.ExternalServiceKind, ExternalServ
   // See the list of parameters at:
   // https://docs.atlassian.com/bitbucket-server/rest/6.1.2/bitbucket-rest.html#idp355
   "repositoryQuery": [
-  //   "?name=<repo>\u0026projectname=<project>" // set this to "none" to disable querying
+  //   "?" // An empty query string means no filters, so it matches all repos visible to the configured token.
```

Comment:
I find it a bit confusing calling this an "empty query string". The field is "repositoryQuery", so I instantly thought you meant the empty string. Maybe we should special case `all`?

Reply:
Agree. I think in both sentences it would be clearer if it say

```suggestion
  //   "?" // A single `?` means no filters, so it matches all repos visible to the configured token.
```